src/Parser.php

```diff
@@ -48,76 +46,100 @@ class Parser
 
 					# Others start #
 						(?>T # Literal T
-						(?<hourNum>2[0-3]|[01][0-9]):
-						(?<minuteNum>[0-5][0-9]):
-						(?<secondNum>[0-5][0-9])
+						(?<hour>2[0-3]|[01][0-9]):
+						(?<minute>[0-5][0-9]):
+						(?<second>[0-5][0-9])
```

Comment:
This is the opposite of what you did in https://github.com/ProfessionalWiki/EDTF/commit/d3c0268be34466876480a04c6db1bb014acec7c3. Did you accidentally do a partial revert?

Reply:
No, I change this value so it can be easier to use in this [loop](https://github.com/ProfessionalWiki/EDTF/pull/5/files#diff-47bc8653b21716576958e25b6d7356ecb0f0070f17554d12d2bee985ac211b26R83-R91)